npm/pkg/dataplane/dataplane_windows.go

```diff
@@ -349,8 +357,8 @@ func (dp *DataPlane) setNetworkIDByName(networkName string) error {
 }
 
 func (dp *DataPlane) getAllEndpointIDs() []string {
-	endpointIDs := make([]string, 0, len(dp.endpointCache))
-	for _, endpoint := range dp.endpointCache {
+	endpointIDs := make([]string, 0, len(dp.endpointCache.cache))
```

Comment:
epCache lock unlock here.

Reply:
this is used only at bootup, so no need to use locks